tikv-ctl: Add more information for compact and remove-fail-stores (#5551)
pingcap/docs: tikv-control.md

- Use the `--host` option to specify the TiKV that you need to compact.
- Use the `-d` option to specify the RocksDB that you need to compact. The optional values are `kv` and `raft`.
- Use the `--threads` option allows you to specify the concurrency that you compact and its default value is 8. Generally, a higher concurrency comes with a faster compact speed, which might yet affect the service. You need to choose an appropriate concurrency based on the scenario.

> - Use the `--bottommost` option to include or exclude the bottommost file when you compact. The option values are `default`、`skip` and `force`. The default value is `default`.

Reviewer: ```suggestion
- Use the `--bottommost` option to include or exclude the bottommost files when TiKV performs compaction. The value options are `default`, `skip`, and `force`. The default value is `default`.
```

>     - `default` means that the bottommost file is included only when you enable the Compaction Filter feature.

Reviewer: ```suggestion
    - `default` means that the bottommost files are included only when the Compaction Filter feature is enabled.
```

>     - `skip` means that the bottommost file is exclueded when you compact.

Reviewer: ```suggestion
    - `skip` means that the bottommost files are excluded when TiKV performs compaction.
```

>     - `force` means that the bottommost file is always included when you compact.

Reviewer: ```suggestion
    - `force` means that the bottommost files are always included when TiKV performs compaction.
```

> **Note:**
>
> - This command only supports the local mode. It prints `success!` when successfully run.

> > - You must run this command for all stores where specified Regions' peers are located. If `-r` is not set, all Regions are involved, and you need to run this command for all stores. You need to restart the healthy stores before restoring them. Otherwise, the inconsistency of the peer lists in Region replicas will cause an error when you execute `split-region` or `remove-peer`, resulting in the inconsistency of other metadata, and finally the Regions will become unavailable.

Reviewer: @Connor1996 There is an evident inconsistency between the Chinese and the English for this note. Please help determine which version prevails. Thanks!

- The Chinese version: "如果使用 `--all-regions`，通常需要在集群剩余所有健康的 store 上执行此命令。"
- The English version: "You must run this command for all stores where specified Regions' peers are located. If `-r` is not set, all Regions are involved, and you need to run this command for all stores."
```suggestion
> - You must run this command for all stores where specified Regions' peers are located. If `-r` is not set, all Regions are involved, and you need to run this command for all stores. You need to ensure that the healthy stores stop providing services before restoring these stores. Otherwise, the inconsistent peer lists among Region replicas will cause an error when you execute `split-region` or `remove-peer`, which causes inconsistency between other metadata, and finally, the Regions will become unavailable.
```

Reviewer 2: The Chinese version prevails.

Reviewer: @Liuxiaozhen12 Please address this comment and align the first sentence with the Chinese. Thanks!

> **Note:**
>
> - This command only supports the local mode. It prints `success!` when successfully run.
> - You must run this command for all stores where specified Regions' peers are located.

> > - If you use the `--all-regions` command, usually you need to run this command for all the remaining healthy stores in the cluster. You need to ensure that the healthy stores stop providing services before restoring these stores. Otherwise, the inconsistent peer lists among Region replicas will cause an error when you execute `split-region` or `remove-peer`, which causes inconsistency between other metadata, and finally, the Regions will become unavailable.

Reviewer: ```suggestion
> - If the `--all-regions` option is used, usually you need to run this command for all the remaining healthy stores in the cluster. You need to ensure that the healthy stores stop providing services before recovering the damaged stores. Otherwise, the inconsistent peer lists in Region replicas will cause errors when you execute `split-region` or `remove-peer`. This further causes inconsistency between other metadata, and finally, the Regions will become unavailable.
```
